include/error/formatters/error_formatters.js

```diff
@@ -103,12 +112,53 @@ module.exports = function(pb) {
      * @param {Function} cb
      */
     ErrorFormatters.html = function(params, cb) {
-        cb(
-            null,
-            '<html><body><h2>Whoops! Something unexpected happened.</h2><br/><pre>' +
-            params.error.stack +
-            '</pre></body></html>'
-        );
+        if (params.errorCount > 1) {
+            
+            //we know we've hit a recursive error situation.  Bail out
+            return cb(
+                null,
+                '<html><body><h2>Whoops! Something unexpected happened.</h2><br/><pre>' +
+                params.error.stack +
+                '</pre></body></html>'
+            );
+        }
```

Comment:
when we throw an error in our error controller we want to catch it and default it.